src/main/java/duke/parser/ParserDuke.java

```diff
@@ -0,0 +1,134 @@
+                    System.out.println("'Tis a new sky for you to scale! Here! \n" + newItem.toString()
+                            + "\nYou now have " + listOfItems.knowTaskCount() + " tasks to do!"
+                            + "\n***********************************************************************");
+                }
+            } else if (command.contains(DEADLINE_CMD)) {
+                String deadline1 = command.replaceAll("deadline ", "");
+                String[] words = deadline1.split("/");
+                String task = words[0];
+                String deadline = words[1];
+                if (!task.isEmpty()) {
+                    ListObject newItem = new Deadline(task, 0, deadline);
+                    listOfItems.handleItemAddition(newItem);
+                    Storage.makeListFile(FILE_PATH, listOfItems);
+                    System.out.println("Mark this on your calendar! \n" + newItem.toString()
+                            + "\nYou now have " + listOfItems.knowTaskCount() + " tasks to do!"
+                            + "\n***********************************************************************");
+                } else if (task.isEmpty()) {
+                    System.out.println("The folly of youth to cheat Time! Write your task following 'deadline'");
+                    System.out.println("***********************************************************************");
+                }
+            } else if (command.contains(EVENT_CMD)) {
+                String event1 = command.replaceAll("event ", "");
+                String[] words = event1.split("/");
+                String task = words[0];
+                String event = words[1];
+                if(!task.isEmpty()) {
+                    ListObject newItem = new Event(task, 0, event);
+                    listOfItems.handleItemAddition(newItem);
+                    Storage.makeListFile(FILE_PATH, listOfItems);
+                    System.out.println("Another moment to mark... \n" + newItem.toString()
+                            + "\nYou now have " + listOfItems.knowTaskCount() + " tasks to do!"
+                            + "\n***********************************************************************");
+                } else if (task.isEmpty()) {
+                    System.out.println("Come Alive! Write an activity following 'event'");
+                    System.out.println("***********************************************************************");
+                }
+            } else if (command.contains(FIND_CMD)) {
+                String target = command.replaceAll("find ", "");
+                System.out.println("Here is what you are looking for!");
+
+            } else {
+                System.out.println("Why trouble me with the unrefined language of the youth! Speak plainly, my friend!");
+                System.out.println("***********************************************************************");
+            }
```

Comment:
Perhaps it may be better to abstract out some of the logic so that the function is not too long?

Reply:
Yes...perhaps a subclass of the Parser dealing with String manipulation on targets might help. Thanks for the suggestion!